File: baseble/src/main/java/com/fcbox/ble/baseble/callback/scan/IScanCallback.java
package com.fcbox.ble.baseble.callback.scan;

import com.fcbox.ble.baseble.model.BluetoothLeDevice;
import com.fcbox.ble.baseble.model.BluetoothLeDeviceStore;


/**
 * 扫描回调
 */
public interface IScanCallback {
    //发现设备
    void onDeviceFound(BluetoothLeDevice bluetoothLeDevice);

    //扫描完成
    void onScanFinish(BluetoothLeDeviceStore bluetoothLeDeviceStore);

    //扫描超时
    void onScanTimeout();

}

File: baseble/src/main/java/com/xzy/ble/baseble/callback/scan/IScanCallback.kt
package com.xzy.ble.baseble.callback.scan

import com.xzy.ble.baseble.model.BluetoothLeDevice
import com.xzy.ble.baseble.model.BluetoothLeDeviceStore


/**
 * 扫描回调
 */
interface IScanCallback {
    //发现设备
    fun onDeviceFound(bluetoothLeDevice: BluetoothLeDevice)

    //扫描完成
    fun onScanFinish(bluetoothLeDeviceStore: BluetoothLeDeviceStore)

    //扫描超时
    fun onscanBleTimeout()

}
